src/ios/sitewaerts/SitewaertsDocumentViewer.m

```diff
@@ -128,6 +128,11 @@ - (void)viewDocument:(CDVInvokedUrlCommand*)command {
                 readerViewController.delegate = self;
                 readerViewController.modalTransitionStyle = UIModalTransitionStyleCrossDissolve;
                 readerViewController.modalPresentationStyle = UIModalPresentationFullScreen;
+                NSInteger pageNumber = [[viewerOptions objectForKey:@"page"] integerValue];
+                if (pageNumber > 1) {
```

Comment:
Here we should have `pageNumber >= 1`
In the current form if we put `{page: 1}` this code will not be executed. 
In my case the PDF was always opened at the page index of the previously opened PDF. I always needed to open the PDF at the first page